sandbox/src/routes/r4/retrieveBusinessFunctions.js

```diff
@@ -16,7 +16,7 @@ module.exports = [
       }
 
       // this should never happen as we always get a valid response for this endpoint
-      return h.file('../SandboxErrorOutcome.json').code(400);
```

Comment:
Nice one, I think if this statement was executed it would have thrown a 500 as well, because the path was wrong

Reply:
In fact, I think we need to change these paths now, because when we restructured the repo, we introduced mocks/r4 and mocks/stu3. When we specifu h.file this is relative to the mocks/ dir - see `sandbox/src/app.js` :
```
      files: {
        relativeTo: Path.join(__dirname, 'mocks'),
      },
```
I've tested locally with `h.file('r4/R4-SandboxErrorOutcome.json').code(400)` and it works fine. We'd have to do the same for every other sandbox endpoint. In most cases the error statement is not reached, so it's been camouflaged for a while.

Reply:
Good spot! fixed :)